addons/mail/models/mail_mail.py

```diff
@@ -293,11 +293,11 @@ def _send_prepare_values(self, partner=None):
             emails_normalized = tools.email_normalize_all(partner.email)
             if emails_normalized:
                 email_to = [
-                    tools.formataddr((partner.name or "False", email or "False"))
+                    tools.contact_formataddr((partner.name or "False", email or "False"))
```

Comment:
> At(@) symbol is removed before the emails are sent

Can you improve a bit the commit message ? (we just remove the @ from the name part, etc)